src/SourceBuild/Arcade/tools/SourceBuildArcadeTarball.targets

```diff
@@ -6,23 +6,33 @@
   <UsingTask TaskName="Microsoft.DotNet.SourceBuild.Tasks.Tarball_ReadSourceBuildIntermediateNupkgDependencies" AssemblyFile="$(SourceBuildTasksAssembly)" />
   <UsingTask TaskName="Microsoft.DotNet.SourceBuild.Tasks.Tarball_WriteSourceRepoProperties" AssemblyFile="$(SourceBuildTasksAssembly)" />
 
+  <PropertyGroup>
+    <TarballDir>$(RepoRoot)artifacts/tarball/</TarballDir>
+    <TarballRootDir>$([MSBuild]::EnsureTrailingSlash('$(TarballDir)'))</TarballRootDir>
+    <TarballSourceDir>$(TarballRootDir)src/</TarballSourceDir>
+    <TarballGitInfoDir>$(TarballRootDir)git-info/</TarballGitInfoDir>
+    <CloneVerbosity>quiet</CloneVerbosity> <!-- Support quiet and full -->
+  </PropertyGroup>
+
   <Target Name="CreateSourceTarball"
     DependsOnTargets="
       CreateTarballDir;
       SetupSelfGithubInfo;
       CloneRepoAndDependentsRecursive;
       CopyTarballContent;
       ">
-      <Message Text="Tarball successfully created: $(TarballRootDir)" Importance="High" />
-    </Target>
 
-  <PropertyGroup>
-    <TarballDir>$(RepoRoot)artifacts/tarball/</TarballDir>
-    <TarballRootDir>$([MSBuild]::EnsureTrailingSlash('$(TarballDir)'))</TarballRootDir>
-    <TarballSourceDir>$(TarballRootDir)src/</TarballSourceDir>
-    <TarballGitInfoDir>$(TarballRootDir)git-info/</TarballGitInfoDir>
-    <CloneVerbosity>quiet</CloneVerbosity> <!-- Support quiet and full -->
-  </PropertyGroup>
+    <PropertyGroup>
+      <TarballFileDir>$([System.IO.Path]::GetDirectoryName('$(TarballFilePath)'))</TarballFileDir>
+    </PropertyGroup>
+
+    <Error Text="TarballFilePath not specified." Condition=" '$(TarballFilePath)' == '' " />
+
+    <MakeDir Directories="$(TarballFileDir)" Condition=" '$(TarballFileDir)' != '' " />
+    <Exec Command="tar --numeric-owner -zcf $(TarballFilePath) $(TarballDir)" />
```

Comment:
In previous .NET versions, the tarball content was nested in a dotnet-<version> directory.  Is this necessary going forward?  It is inconsistent with the other published .NET tarballs.  It feels like excluding it gives the consumer more freedom to easily extract the content into a folder of their choosing.

cc @omajid

Reply:
I'm not sure it matters that it is in a dotnet- versioned directory as long as it is consistent, which it will be with building it in installer.  However, looking at the tarball from this CI build, it includes folders from the CI system.  So when I unpack it, it unpacks into a `__w/1/s/artifacts/tarball` folder.  It'd be nice if the inner folder was just `tarball` or `.`

Reply:
That was not intentional.  I have addressed this.